packages/synapse-react-client/stories/SynapseFormSubmissionsGrid.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { Meta, StoryObj } from '@storybook/react'
+import { rest } from 'msw'
+import { MOCK_REPO_ORIGIN } from '../src/lib/utils/functions/getEndpoint'
+import SynapseFormSubmissionGrid from '../src/lib/containers/synapse_form_wrapper/SynapseFormSubmissionsGrid'
+import { SynapseContextConsumer } from '../src/lib/utils/SynapseContext'
+import FullContextProvider from '../src/lib/utils/FullContextProvider'
+import { ListRequest, StatusEnum } from '../src/lib/utils/synapseTypes'
+import {
+  formListDataInProgress,
+  formListDataSubmitted,
+} from '../mocks/mock_drug_tool_data'
+import { getHandlers } from '../mocks/msw/handlers'
+
+const meta: Meta = {
+  title: 'Portals/SynapseFormSubmissionsGrid',
```

Comment:
Happy to move this story, if there's a better location for it!

Reply:
This is great. Thanks for adding this!